Library/GrantedBy.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+    /// <summary>
```

Comment:
This doesnt seem to be doing anything, what about taking your ReadXMLGrantedBys method above and removing it from Entitlement and implementing it in this class instead?